circle.yml

```diff
@@ -44,6 +46,8 @@ deployment:
       - TESTPILOT_BUCKET=testpilot.dev.mozaws.net ./bin/deploy.sh dev
       - aws configure set preview.cloudfront true
       - aws cloudfront create-invalidation --distribution-id E2ERG47PHCWD0Z --paths '/*'
+      - docker pull owasp/zap2docker-weekly
+      - timeout 300 docker run -t owasp/zap2docker-weekly zap-baseline.py -t https://testpilot.dev.mozaws.net/
```

Comment:
It's technically been deployed, but the CDN doesn't know about it until later, and the CDN + nginx proxy that sits between the CDN and our S3 bucket are important for adding security headers.

Our build does not currently know when the CDN invalidation has been finished. You'd need to poll the URL returned by the command on `aws cloudfront create-invalidation`, monitoring for `status` to change to `Completed`.

Is it possible to run this asynchronously after a build has completed, kicking off by HTTP request or similar, and reporting results in IRC? I'd probably push back on any attempt to wait for the invalidation to finish, as it would likely double our build times.